contracts/bonding/BondingManager.sol

```diff
@@ -376,9 +379,9 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
         UnbondingLock storage lock = del.unbondingLocks[_unbondingLockId];
 
         // Unbonding lock must be valid
-        require(isValidUnbondingLock(msg.sender, _unbondingLockId));
+        require(isValidUnbondingLock(msg.sender, _unbondingLockId), "invalid unbonding lock ID");
         // Withdrawal must be valid for the unbonding lock i.e. the withdraw round is now or in the past
-        require(lock.withdrawRound <= roundsManager().currentRound());
+        require(lock.withdrawRound <= roundsManager().currentRound(), "withdraw round must be equal to or smaller than the current round");
```

Comment:
Let's try to use the same language as the other revert reasons when comparing values (I see that you're using the language "before or equal to"). This reason could be:

"withdraw round must be before or equal to the current round"

Reply:
fixed in 2eb6a0f